modules/postgresql/variables.tf

```diff
@@ -115,7 +115,10 @@ variable "authorized_gae_applications" {
 
 variable "ip_configuration" {
   description = "The ip configuration for the master instances."
-  default     = {}
+
+  default = {
+    ipv4_enabled = "true"
+  }
```

Comment:
I think this fixes a broken test potentially, but is it on purpose?
In my opinion, this looks reasonable because our official document says:

```
The settings for IP Management. This allows to enable or disable the instance IP and manage which external networks can connect to the instance. The IPv4 address cannot be disabled for Second Generation instances.
```
https://cloud.google.com/sql/docs/mysql/admin-api/v1beta4/instances/insert#request-body

What do you think?

Reply:
This actually fixes the module being broken entirely:
```
ipv4_enabled - (Optional) Whether this Cloud SQL instance should be assigned a public IPV4 address. Either ipv4_enabled must be enabled or a private_network must be configured.
```
Passing in an empty map leads to an error due to Terraform expecting one of these values to be set.

Reply:
Yep I know, but previous implementation doesn't raise an error.
I think current implementation looks fine, thanks.